airflow/contrib/hooks/gcp_cloudml_hook.py

```diff
@@ -0,0 +1,138 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import logging
+import time
+from apiclient.discovery import build
+from apiclient import errors
+from oauth2client.client import GoogleCredentials
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+from airflow import settings
+
+logging.getLogger('GoogleCloudML').setLevel(settings.LOGGING_LEVEL)
+
+
+class _CloudMLJob(object):
+    """CloudML job operations helper class."""
+
+    def __init__(self, cloudml, project_name, job_id, job_spec=None):
+        assert project_name is not None and project_name is not ''
```

Comment:
Should compared against '' with `!=` since logically we are checking equality, not identity. I'm not even sure that '' is guaranteed to be interned.